api/python/provisioner/commands/deploy.py

```diff
@@ -83,22 +84,31 @@
     iopath=[
         "misc_pkgs.lustre",
         "motr",
-        "s3server"
+        "s3server",
+        "hare"
     ],
+    # The R1 HW HA sequence
+    # ha=[
+    #     "ha.corosync-pacemaker",
+    #     "ha.haproxy.start",
+    #     "hare",
+    #     "ha.cortx-ha",
+    #     "ha.iostack-ha"
+    # ],
+
     ha=[
-        "ha.corosync-pacemaker",
+        "ha.corosync-pacemaker.install",
+        "ha.corosync-pacemaker.config.base",
         "ha.haproxy.start",
```

Comment:
haproxy is required to be running for s3 as well. Better move this to prepreqs list